### PR TITLE
Streamline image build in vast-cloud

### PR DESCRIPTION
**Pull request by rdettai**
The current mechanism to build docker images for the cloud environment is hard to extend for new plugins (e.g the misp integration. 

The way that was retained is to use docker compose to coordinate the build of images plugin by plugin. This allows us to:
- forward env to build args
- locally define the context and service name

### :memo: Reviewer Checklist

Review this pull request by ensuring the following items:

- [x] All user-facing changes have changelog entries
- [x] User-facing changes are reflected on [vast.io](https://github.com/tenzir/vast/tree/master/web)

Reviewers should also note that they:

- should rebuild the CLI before testing this change (`VASTCLOUD_REBUILD=1 ./vast-cloud`)

[Terraform](https://github.com/tenzir/vast/actions/workflows/terraform.yaml)

### CODE REVIEW

**Comment by KaanSK** [comment]
Had issues with dockerized aws client and after discussing with @rdettai we solved the issue. Dockerized AWS client file permissions on credentials file should be set properly. Just adding this as comment on review process. No further action is required.

**Comment by KaanSK** [comment]
Would it make sense to add instruction to use `./vast-cloud init` for terraform required dependencies (I guess they are called external modules/libraries)? While reviewing this, I needed to use it.

**Comment by rdettai** [comment]
> Dockerized AWS client file permissions on credentials file should be set properly.

To be more precise, the credentials file should be readable by the user executing `./vast-cloud`

> Would it make sense to add instruction to use ./vast-cloud init for terraform required dependencies (I guess they are called external modules/libraries)? While reviewing this, I needed to use it.

Yes I think a "troubleshooting" section would be very helpful

**Comment by rdettai** [comment]
https://github.com/tenzir/vast/actions/runs/2963130323

**Comment by rdettai** [comment]
@KaanSK I think I addressed both your comments:
- destroy failing when core modules not applied
- added comment abount init command

**Review by KaanSK — CHANGES_REQUESTED** [review]

File: cloud/aws/cli/core.py [hunk 1]
```diff
@@ -64,6 +64,11 @@ def active_include_dirs(c: Context) -> str:
     return " ".join([f"--terragrunt-include-dir={mod}" for mod in active_modules(c)])
 
 
+def docker_compose(step):
```
[the remark this answers is not in the document]
Comment (KaanSK) [reply cloud/aws/cli/core.py:65]: The review comments were before the https://github.com/tenzir/vast/pull/2536#issuecomment-1232760392 . @rdettai fixed the issue. No further action is required.

File: cloud/aws/terraform/core-2/terragrunt.hcl [hunk 2]
```diff
@@ -2,12 +2,9 @@ include "root" {
   path = find_in_parent_folders("terragrunt.${get_env("TF_STATE_BACKEND")}.hcl")
```
[the remark this answers is not in the document]
Comment (KaanSK) [reply cloud/aws/terraform/core-2/terragrunt.hcl:2]: The review comments were before the https://github.com/tenzir/vast/pull/2536#issuecomment-1232760392 . @rdettai fixed the issue. No further action is required.

**Comment by KaanSK** [comment]
> 

Nothing else from my side. LGTM.

**Review by KaanSK — APPROVED** [review]

**Pull request merged**